src/components/warehouse/inventory/InventorySidePane.js

```diff
@@ -63,24 +89,20 @@ const InventorySidePane = ({
             placeholder="Search Tags"
           />
         </div>
-        <div className="form-group">
+        <div>
           <label htmlFor="select" className="inv-input-header">
             Channels
           </label>
-          <select
-            className="form-control"
-            name="channel"
-            onChange={handleChange}
-          >
-            <option className="hover-col" value="shopify">
-              All Channels
-            </option>
-            <option className="hover-col" value="shopify">
-              Shopify US
-            </option>
-            <option value="fba-amazon">Amazon</option>
-          </select>
+          <Select
+            theme={selectTheme}
+            label="Single select"
+            options={shopOptions}
+            className="mb-3"
+            placeholder="All Channels"
+            onChange={handleChanges}
+          />
```

Comment:
Do we external library for just `select` ? I don't we want to do this.
![image](https://user-images.githubusercontent.com/34476840/89534577-105d6a80-d7ed-11ea-8bd8-9d85c6a54c82.png)